odoo/tools/translate.py

```diff
@@ -456,7 +462,22 @@ def __call__(self, source):
                 cr.close()
         return res
 
+
+class LazyStrTranslator:
+
+    __slots__ = ['_source']
+    def __init__(self, source):
+        self._source = source
+
+    def __repr__(self):
+        return _._get_translation(self._source)
+
+    def __eq__(self, other):
+        raise NotImplementedError()
```

Comment:
Please explain why.

Reply:
Maybe add the decorator `@functools.total_ordering` to override **all** comparison operators.